CHANGELOG.md

```diff
@@ -20,6 +20,16 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Changed
 
 - Jaeger remote sampler's probabilistic strategy now uses the same sampling algorithm as `trace.TraceIDRatioBased` in `go.opentelemetry.io/contrib/samplers/jaegerremote`. (#6892)
+- Switched the default for `OTEL_SEMCONV_STABILITY_OPT_IN` to emit the v1.26.0 semantic conventions by default in the following modules. (#6899)
+  - `go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful`
+  - `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin`
+  - `go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux`
+  - `go.opentelemetry.io/contrib/instrumentation/github.com/labstack/echo/otelecho`
+  - `go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace`
+  - `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`
+
+  The `OTEL_SEMCONV_STABILITY_OPT_IN=http/dup` environment variable can be still used to emit both the v1.20.0 and v1.26.0 semantic conventions.
```

Comment:
Do I understand correctly that with this change it is NOT possible to emit only the 1.20.0 semantic conventions? Basically, this makes `OTEL_SEMCONV_STABILITY_OPT_IN=http` as default, right?

If so then, I think we should explicitly call it out. Especially, given it looks to be against https://opentelemetry.io/docs/specs/semconv/http/

Reply:
> However, that behavior is not documented in the [semconv migration spec](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/README.md),

Maybe, it should be documented? 

@open-telemetry/semconv-http-approvers, any recommendations?

Reply:
Since we're only keeping this behavior for one release, I'm not sure allowing to emit only the 1.20.0 semantic conventions is a good idea. Folks need to migrate, and keeping the old behavior would let them stall.

For folks that ignored this, after the next release, they will automatically get the new semconvs.
If that breaks their behavior, they can get both while they migrate away their dashboards and alerts.

Reply:
I've added a specific mention.